src/main/resources/db/changelog/20230921144200-add_reg_auth_method_to_cp_consumers.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd">
+
+    <changeSet id="20230021144200-1" author="wpoteat">
+        <comment>
+            Add registration authorization method to the cp_consumer table.
+        </comment>
+
+        <addColumn tableName="cp_consumer">
+            <column name="reg_auth_method" type="VARCHAR(255)"/>
```

Comment:
This column should probably be nullable (all the existing rows will need to have null as the default value).

Reply:
Also, we can probably save some space/performance by reducing the varchar from 255 chars to 30 or so. Its an enum, and I doubt we will ever add a new auth type with a name longer than that. If we do, we can always abbreviate it.